lib/rubocop/extension/generator/generator.rb

```diff
@@ -145,8 +145,11 @@ def rules(_context)
             gem 'rspec'
           RUBY
 
+          # Bundler's generated README.md has had 3 styles of gem declarations.
           if Gem::Version.new(Bundler::VERSION) >= Gem::Version.new('2.3.9')
             patch 'README.md', /\$ bundle add (.*)$/, '$ bundle add \1 --require=false'
+          if Gem::Version.new(Bundler::VERSION) >= Gem::Version.new('2.6.0')
+            patch 'README.md', /^\s*(?:\$\s*)?bundle add[^\n]*$/, '\\0 --require=false'
```

Comment:
It looks like `2.3.9`+ would be processed first unless `2.6.0`+ comes before it. Also, it needs to be `elsif` :-)

```suggestion
          if Gem::Version.new(Bundler::VERSION) >= Gem::Version.new('2.6.0')
            patch 'README.md', /^\s*(?:\$\s*)?bundle add[^\n]*$/, '\0 --require=false'
          elsif Gem::Version.new(Bundler::VERSION) >= Gem::Version.new('2.3.9')
            patch 'README.md', /\$ bundle add (.*)$/, '$ bundle add \1 --require=false'
```

Alternatively, I think the following could be considered as a compatibility patch for both `2.3.9`+ and `2.6.0`+.


```ruby
          # Bundler's generated README.md has had two styles of gem declarations.
          # See: https://github.com/rubygems/rubygems/commit/c805e9b558
          if Gem::Version.new(Bundler::VERSION) >= Gem::Version.new('2.3.9')
            patch 'README.md', /^\s*(?:\$\s*)?bundle add[^\n]*$/, '\0 --require=false'
          else
            patch 'README.md', /^gem '#{name}'$/, "gem '#{name}', require: false"
          end
```

Reply:
🤦 sorry man, I shouldn't try to write changes from my phone before I've had my coffee. Force pushed again and hopefully didn't screw it up a third time.

Reply:
Yeah, looks good! Thank you for catching up!